supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc

```diff
@@ -122,13 +155,15 @@
 [discrete]
 [[trusted-certificate-authority]]
 ==== image:images/yes.png[yes] trusted Certificate Authority (noun)
-*Description*: A "trusted Certificate Authority" is a third party that creates SSL certificates (CA certificates) used for authentication. It is not to be confused with self-signed certificates. Note the capitalization of Certificate Authority, commonly abbreviated as CA.
+*Description*: A trusted Certificate Authority (CA) is a third-party entity that creates TLS certificates, known as CA certificates, for authentication purposes. A trusted CA is different from a self-signed certificate; in that, a self-signed certificate has its own private key and does not need to request a key from a public or private CA. Note the capitalization of Certificate Authority, which is commonly abbreviated as CA.
```

Comment:
```suggestion
*Description*: A trusted Certificate Authority (CA) is a third-party entity that creates TLS certificates, known as CA certificates, for authentication purposes. A trusted CA is different from a self-signed certificate in that a self-signed certificate has its own private key and does not need to request a key from a public or private CA. Note the capitalization of Certificate Authority, which is commonly abbreviated as CA.
```
Omit semicolon and comma. Per IBM SG, only use semicolons to separate independent clauses. In the current version, the semicolon is being used to separate the first independent clause from the second dependent clause (the one that begins with `in that...`)